src/configuration/defaults.ts

```diff
@@ -7,6 +7,8 @@ export const defaultModesWeSellTicketsFor: string[] = [
   'localTram',
   'regionalBus',
   'shuttleBus',
+  'highSpeedPassengerService',
+  'highSpeedVehicleService',
```

Comment:
defaultModesWeSellTicketsFor should also be reused here:

https://github.com/AtB-AS/mittatb-app/blob/11e160b6bb6f6731eb6b87285c6219a686593dfd/src/travel-details-screens/TripDetailsScreenComponent.tsx#L194-L201

However, since all the elements in the list are actually submodes, this variable is incorrectly named. Not a part of your changes, but I think it is a good opportunity to correct it.
Should rename modesWeSellTicketsFor in FirestoreConfigurationContext as well.

Reply:
Suggested renaming:
defaultModesWeSellTicketsFor -> defaultSubModesWeSellTicketsFor
modesWeSellTicketsFor -> subModesWeSellTicketsFor

Reply:
defaultModesWeSellTicketsFor should not be used there, as is mentioned in the comments above:
https://github.com/AtB-AS/mittatb-app/blob/11e160b6bb6f6731eb6b87285c6219a686593dfd/src/travel-details-screens/TripDetailsScreenComponent.tsx#L192-L193

Reply:
Good catch! However still add 'highSpeedPassengerService', 'highSpeedVehicleService'

Reply:
> Suggested renaming:
> defaultModesWeSellTicketsFor -> defaultSubModesWeSellTicketsFor
> modesWeSellTicketsFor -> subModesWeSellTicketsFor

This would have to be changed in firestore as well, and would be a bigger change

Reply:
I guess it can be part of this issue, or a new one. Your call :+1:

Reply:
> Good catch! However still add 'highSpeedPassengerService', 'highSpeedVehicleService'

We do not currently support puchasing boat tickets from trip details, and as of now only supports regular single tickets

Reply:
Fair enough :+1: